### ISSUE TITLE
Accessibility: Send button does not have a label for Voice Control

### ISSUE THREAD

**Issue by taowa** [signalapp/Signal-iOS#4991]
- [ x ] I have searched open and closed issues for duplicates
- [ x ] I am submitting a bug report for existing functionality that does not work as intended
- [ x ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
The send button does not have a label that Voice Control can use.

### Steps to reproduce
- enable voice control (settings -> accessibility -> voice control)
- say "show numbers continuously" or "show names continuously"
- open a conversation
- tap the box to enter text

**Actual result:** a number or name does not appear over the send button (see first three photos)

**Expected result:** a number or name appears over the send button (see last photo)

### Screenshots
![signal-1](https://user-images.githubusercontent.com/1357889/117384755-b12ea280-aeb1-11eb-9fcb-8d09d8f3f598.jpeg)
![signal-2](https://user-images.githubusercontent.com/1357889/117384763-b4299300-aeb1-11eb-80cc-3625489c8ae5.jpeg)
![signal-3](https://user-images.githubusercontent.com/1357889/117384765-b7248380-aeb1-11eb-9127-ea35dae65dce.jpeg)
![notsignal-1](https://user-images.githubusercontent.com/1357889/117384774-bab80a80-aeb1-11eb-891b-1a14f6e418a7.jpeg)

### Device info
**Device**: iPhone 7

**iOS version**: 14.5.1

**Signal version:** 5.10.0.49

**Comment by haniewais** [comment]
How was this issue fixed?
